app/controllers/new_administrateur/procedures_controller.rb

```diff
@@ -186,7 +186,12 @@ def transfer
     end
 
     def invited_expert_list
-      @invited_expert_emails = ExpertsProcedure.invited_expert_emails(@procedure)
+      @experts_procedure = ExpertsProcedure.invited_experts(@procedure)
+    end
+
+    def toggle_allow_decision_access
+      @expert_procedure = ExpertsProcedure.find(params[:expert_procedure])
+      @expert_procedure.toggle!(:allow_decision_access)
```

Comment:
1. J'ai l'impression que tu ne vérifies pas qui a la droit de modifier l accès a la décision, c'est a dire que n'importe qui avec la bonne requête pourrait activer cette feature
2. Pourquoi fais tu une affectation ?

```suggestion
      @procedure
        .experts_procedures
        .find(params[:expert_procedure])
        .toggle!(:allow_decision_access)
```